tests/utils/test_Files.py

```diff
@@ -191,16 +189,34 @@ def test_folder_zip(self):
 
         assert zip_file_list(zip_file) == ['temp_file.txt']
 
-
     def test_path_combine(self):
-        assert path_combine('a', 'b') == f"{path_current()}/a/b"       # todo: add more use cases
+        assert path_combine('a', 'b') == f"{path_current()}/a/b"  # todo: add more use cases
```

Comment:
Sorry can you revert back all those changes (i.e. all the spaces you removed)

the idea with that type of alignment is to make it easier to read

Reply:
and making sure that items are aligned in a way that the eyes can easily consume the information/code